src/test/scala/com/snowflake/snowpark/ParameterSuite.scala

```diff
@@ -65,4 +69,19 @@ class ParameterSuite extends SNTestBase {
       ParameterUtils
         .jdbcConfig(optionWithoutKey + ("privatekey" -> "wrong key"), isScalaAPI = true))
   }
+
+  test("enable to read PKCS#8 private keys") {
```

Comment:
Can you confirm we have test case to cover for both RSA and PKCS1

Reply:
Yes, both `pkcs1` and `pkcs8`